src/renderer/html_handlebars/search.rs

```diff
@@ -71,11 +71,21 @@ fn render_item(
     item: &BookItem,
 ) -> Result<()> {
     let chapter = match *item {
-        BookItem::Chapter(ref ch) => ch,
+        BookItem::Chapter(ref ch) => {
+            if let Some(_) = ch.path {
+                ch
+            } else {
+                return Ok(());
+            }
+        }
```

Comment:
Can probably be simplified.

```suggestion
        BookItem::Chapter(ref ch) if !ch.is_draft_chapter() => ch,
```